uiowa/uiowa
# Feature: Banner size, gradient, play button updates

# How to test

```
rm -rf ./node_modules
yarn cache clean
yarn install
yarn workspace uids_base gulp --development
blt ds --site=admissions.uiowa.edu
```

## Test aspect ratio size and gradient

1. Check that banners resize and no longer stay at a fixed size. 
2. Check that banner gradient remains applied in browser width drops below container breakpoint.
3. Go to https://admissions.local.drupal.uiowa.edu/student-profiles/aasthika-das and verify that banner gradient has been removed. 

## Test video play button

1. `blt ds --site=iti.uiowa.edu`  
2. Verify that video button no longer overlaps with the link button on the home page of that site
3. Verify that video at the bottom of https://iti.local.drupal.uiowa.edu/our-partnerships does have a play button overlap issue like https://iti.uiowa.edu/our-partnerships
